src/components/deploymentConfig/DeploymentConfig.tsx

```diff
@@ -750,15 +610,8 @@ export default function DeploymentConfig({
             case 1:
             case 3:
                 setIsValues(true)
```

Comment:
Please verify if this is needed at the time of tabChange

Reply:
This is required since on tab switch to (Compare) we need to switch yamlMode to true. Thus on switching back, to keep the last state added a ref to store the previous value as per discussion.